src/map/atcommand.c

```diff
@@ -1704,6 +1704,32 @@ ACMD(model)
 }
 
 /*==========================================
+ * @bodystyle [Rytech]
+ *------------------------------------------*/
+ACMD(bodystyle)
+{
+	int body_style = 0;
+
+	memset(atcmd_output, '\0', sizeof(atcmd_output));
+
+	if (!message || !*message || sscanf(message, "%d", &body_style) < 1) {
```

Comment:
`message` is guaranteed by contract to be non-NULL. You should remove the `!message` condition from here